gusto/core/configuration.py

```diff
@@ -200,8 +200,9 @@ class SUPGOptions(WrapperOptions):
 
     name = "supg"
     tau = None
-    default = 1/sqrt(15)
+    default = 1/(sqrt(15))
     ibp = IntegrateByParts.TWICE
+    field_names = []
```

Comment:
It might be more natural to have `None` as the default option here

e.g. : https://nikos7am.com/posts/mutable-default-arguments/#:~:text=The%20solution%20and%20the%20standard%20way%20of%20doing,list_to_append.%20append%28element%29%20return%20list_to_append%20%3E%3E%3E%20b%20%3D%20append_to_list%2820%29

Reply:
Done!